src/App.css

```diff
@@ -1,6 +1,7 @@
-@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@500&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@500&display=swap');
```

Comment:
Looks like the linter hit this file when it wasn't before? Wonder why that is...

Reply:
This is weird, but I did remember VSCode prompted me about the linter. So maybe it's the first time that it started kicking in... Super weird though because the CSS file was definitely linted.

May also have something to do with the `prettier` update?